### DISCUSSION TITLE
大图渲染失败，程序crash

### DISCUSSION THREAD

**Discussion by l540144279** [Tencent/libpag#1826] [Q&A] [answered]
Linux，偶尔较大尺寸或分辨率的图片，渲染上屏时会导致程序crash，想知道pag图片渲染的最大限制是什么？ 内存、CPU占用或者哪个维度的限制，是否有设置选项api控制？

**Comment by kevingpqi123** [comment, marked as answer]
pag 对于图片渲染没有限制，此类问题建议提供完整的  crash 信息，仅仅靠描述无法定位问题
关于服务端渲染，腾讯内部所有的游戏战报基于的都是 pag 的方案，这部分技术已经很成熟了，还没有其它业务反馈过类似问题
